ansible/roles/test/tasks/lag_2.yml

```diff
@@ -72,9 +72,14 @@
 - name: test each lag interface minimum links and rate
   include: single_lag_test.yml
   with_items: lag_facts.names
-  when: test_minlink|bool == true
+  when: lag_facts.lags[item]['po_config']['runner']['min_ports'] is defined
 
 - name: test each lag interface LACP DU rate
   include: single_lag_lacp_rate_test.yml
   with_items: lag_facts.names
-  when: test_rate|bool == true
\ No newline at end of file
+  when: lag_facts.lags[item]['po_config']['runner']['min_ports'] is defined
```

Comment:
The same here for test_rate.
Please fix it and I'll merge

Reply:
done. thanks!